api/admin/controller/library_settings.py

```diff
@@ -319,7 +321,7 @@ def list_setting(self, setting, json_objects=False):
                 if not isinstance(i, list):
                     i = [i]
                 if json_objects:
-                    i = [json.loads(s) for x in i]
+                    i = [json.loads(x) for x in i]
```

Comment:
This bug went unnoticed because we didn't previously have a setting that was a list of JSON-encoded objects (as opposed to a single JSON-encoded object like the geographic settings, or a list of strings).